fastlane/metadata/android/en-US/title.txt

```diff
@@ -1 +1 @@
-Catima - Loyalty Card & Ticket Manager
+Catima — EDC Wallet for Loyalty Cards, Tickets, and Cupons
```

Comment:
I would rather keep the word EDC out of the title, seems too technical and complicated.

Reply:
The thinking is that "EDC wallet" has no current keyword hits, and I think people are searching for it.
The title is just _something_ currently.